travis/travis_install_nightly

```diff
@@ -33,11 +33,24 @@ fi
 
 : ${ODOO_REPO:="odoo/odoo"}  # default value, if not set
 IFS="/" read -a REPO <<< "${ODOO_REPO}"
-ODOO_URL="https://github.com/${REPO[0]}/${REPO[1]}/archive/${VERSION}.tar.gz"
-
-echo "Installing Odoo ${ODOO_URL}"
-wget -nv -O odoo.tar.gz ${ODOO_URL}
-tar -xf odoo.tar.gz -C ${HOME}
+export REMOTE=$(python -c "print '${REPO[0]}'.lower()")
```

Comment:
I prefer use python to have the lines of code for a future migration, but let to go change like as your recommendation.
Thanks
+1